lib/assets/sass/prototype.scss

```diff
@@ -9,3 +9,46 @@
 
 // SASS that is user defined
 @import ".tmp/sass/user/application";
+
+// SASS for error page
+
+.govuk-prototype-kit-error-page-header {
+  .govuk-header__container {
+    border-color: #d4351c;
```

Comment:
This is in `govuk-frontend` as `red` (https://github.com/alphagov/govuk-frontend/blob/839adc06ad36aeb1a982d8fd778b39c3c1ee2bae/packages/govuk-frontend/src/govuk/settings/_colours-palette.scss#L18) I think we should be using that if we can in this context.

Reply:
Now uses the govuk-colour("red") macro